test_tar_changes.py

```diff
@@ -75,3 +75,10 @@ def test_upload_source(delete_sources, mocked_run, mocked_sources):
     # flag was detected by argparse
     tarchanges.upload_source(osdist, 'test.tar', False)
     mocked_run.assert_called_once() # run() was not called a 2nd time
+
+@patch.object(tarchanges,'git')
+def test_diff_filenames(mocked_git):
+    # check if the diff_filenames includes the correct info in git cmds
```

Comment:
i would just change "includes the correct info" to "passes the correct commit range to git" to make it slightly clearer as to what we're actually checking here